src/modules/network.cpp

```diff
@@ -1,5 +1,70 @@
 #include "modules/network.hpp"
 #include <sys/eventfd.h>
+#include <fstream>
+
+namespace {
```

Comment:
As the network module starts to get bigger I was thinking about splitting it into two files: netlink.cpp, network.cpp

Reply:
Yeah that could be better to split out this module
Though I admit I don't know what should go in what file, terminology wise D:

Reply:
I'll take care of it :)